lib/constants/pidParams.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __PIDPARAMS_H__
+#define __PIDPARAMS_H__
+
+//A centralized place to tune the PID
+namespace PidParams {
+  const float Kp = 1;
+  const float Ki = 1;
+  const float Kd = 1;
+};
+#endif
```

Comment:
What do these represent? Are these the specific PID params for a particular PID controller, or is this is a generic struct that can be used whenever you need to pass some PID params somewhere?
If these are specific, then name them as such (ChargeControllerPIDParams or something fitting)
If these are not specific, then make it a struct with the default values 0, 0, 0 (so no error correction).